enhancements/console/customize-catalogs.md

```diff
@@ -0,0 +1,184 @@
+---
+title: customize-catalogs
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@jerolimov"
+  - "@cvogt"
+creation-date: 2022-07-27
+last-updated: 2022-08-03
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Catalogs
+
+## Summary
+
+This document provides details about how cluster administrator 
+can hide developer catalog and sub-catalogs. 
+
+## Motivation
+
+Cluster administrator wants to disable developer catalog or sub-catalogs for all users.
+
+### User Stories
+
+#### Story 1
+
+As a cluster admin, I want to hide Developer catalog for all users.
+
+#### Story 2
+
+As a cluster admin, I want to hide sub-catalogs for all users.
+
+### Goals
+
+- Allow cluster admins to hide developer catalog and sub-catalogs.
+- Provide information to admin about Catalogs and sub-catalogs they can hide from users.
+
+### Non-Goals
+
+N/A
+
+## Proposal
+
+We can extend the existing `operator.openshift.io/v1` / `Console CRD`. 
+We can add 
+`spec.customization.catalog.disabled` which can have list of perspectives disabled.
+
+### Workflow Description
+
+Add the disabled catalogs list along with the sub-catalogs to hide. If types are not
+mentioned then complete catalog will be disabled.
+
+```text
+├── Spec
+│   ├── customization
+|       ├── catalog 
+|           ├──disabled
+|             ├── name
+|             ├── types
+|                 └──         
+└── ...
+```
+
+Each catalog conforms to the following schema:
+
+- `disabled?: authV1.ResourceAttributes[]` , define list of catalogs to hide.
+- `name?: authV1.ResourceAttributes` , defines name of catalog to hide.
+- `types?: authV1.ResourceAttributes[]` , defines a list of sub-catalogs to hide.
```

Comment:
Did you mean `authV1.ResourceAttributes` or something else here?

I'd avoid using TypeScript `?` syntax since this isn't TypeScript code.